simplq/src/components/pages/Status/StatusSidePanel.jsx

```diff
@@ -3,11 +3,13 @@ import SidePanel from '../../common/SidePanel';
 import LeaveQueue from './LeaveQueue';
 import ContactAdmin from './ContactAdmin';
 import QueueDetails from './QueueDetails';
+import NotificationContainer from './NotificationContainer';
 
 export default (props) => (
   <SidePanel>
-    <LeaveQueue />
+    <LeaveQueue leaveQueueHandler={props.leaveQueueHandler} />
     <ContactAdmin />
     <QueueDetails queueId={props.queueId} />
+    <NotificationContainer />
   </SidePanel>
 );
```

Comment:
This is just a personal preference, but I like the styles on the admin sidebar better than this one.

![Screenshot from 2020-10-04 15-55-55](https://user-images.githubusercontent.com/12583528/95013041-29eb2900-065b-11eb-8b44-90c1e8ad09d7.png)


![Screenshot from 2020-10-04 15-55-59](https://user-images.githubusercontent.com/12583528/95013045-2e174680-065b-11eb-8b0e-53e9d8f45069.png)

What are your thoughts @daltonfury42  @akashkbaburajan

Reply:
I think so too. Let's raise a separate issue to discuss this